BusinessLogic/Services/TimeOffRequestService.cs

```diff
@@ -1,12 +1,61 @@
-﻿using DataAccess.Repository.Interfaces;
+﻿using ApiModels.Models;
+using AutoMapper;
+using DataAccess.Repository;
+using DataAccess.Repository.Interfaces;
+using Domain.EF_Models;
+using Domain.Enums;
 using System;
 using System.Collections.Generic;
 using System.Text;
+using System.Threading.Tasks;
 
 namespace BusinessLogic.Services
 {
     public class TimeOffRequestService
```

Comment:
create ITimeOffRequestService interface with appropriate methods, add to DI and inject in the controller